src/elm/Page/Dashboard/Claim.elm

```diff
@@ -287,15 +292,21 @@ viewDetails shared model claim =
                 [ text_ "claim.objective" ]
             , p
                 [ class "pt-2 text-body" ]
-                [ text claim.action.objective.description ]
+                [ text claim.action.objective.description
+                ]
             ]
         , div [ class "mb-8" ]
             [ p
                 [ class "text-caption uppercase text-green" ]
                 [ text_ "claim.your_reward" ]
             , p
                 [ class "pt-2 text-body" ]
-                [ text (String.fromFloat claim.action.verifierReward ++ " " ++ Eos.symbolToSymbolCodeString claim.action.objective.community.symbol) ]
+                [ text
+                    (String.fromFloat claim.action.verifierReward
+                        ++ " "
+                        ++ Eos.symbolToSymbolCodeString selectedCommunity
```

Comment:
same here, selected community might be different.

one way to circumvent this is to have it checked during init and ask/do a community change